### PR TITLE
NOISSUE - Update computation policies docs

### PR DESCRIPTION
**Pull request by smithjilks**

### CODE REVIEW

File: docs/policies.md [hunk 1]
```diff
@@ -44,10 +44,14 @@ Content-Length: 0
 ```
 
 On the UI the steps are as follows:
-1. Login as an Administrator and click on 'Policies' on the side navigation panel. 
-2. Click on 'New Computation Policies' button on the top left.
-![List Policies](img/ui/policies.png)
-3. Select the required details and click 'Create'
+
+1. Login as an Administrator and click on 'Computations' on the side navigation panel.
```
Comment (SammyOina) [inline docs/policies.md:48]: not just admin

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Review by WashingtonKK — CHANGES_REQUESTED** [review]

File: docs/policies.md [hunk 2]
```diff
@@ -44,10 +44,14 @@ Content-Length: 0
...
 On the UI the steps are as follows:
-1. Login as an Administrator and click on 'Policies' on the side navigation panel. 
-2. Click on 'New Computation Policies' button on the top left.
-![List Policies](img/ui/policies.png)
-3. Select the required details and click 'Create'
+
+1. Login as an administrator/computation owner and click on 'Computations' on the side navigation panel.
+![List Computations](img/ui/computations%20list.png)
+2. Select a computation and click on 'Computation Policies'
+![List Policies](img/ui/to computation policies.png)
```
[the remark this answers is not in the document]
Comment (WashingtonKK) [reply docs/policies.md:51]: image

The images do not display on my end.

**Review by WashingtonKK — APPROVED** [review]
LGTM

**Review by SammyOina — CHANGES_REQUESTED** [review]
update the images

File: docs/img/ui/computation policies.png
Comment (SammyOina) [inline docs/img/ui/computation policies.png:1]: remove this file

File: docs/img/ui/computations list.png
Comment (SammyOina) [inline docs/img/ui/computations list.png:1]: same here

File: docs/img/ui/to computation policies.png
Comment (SammyOina) [inline docs/img/ui/to computation policies.png:1]: outdated image

File: docs/img/ui/update policy page.png
Comment (SammyOina) [inline docs/img/ui/update policy page.png:1]: same here

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — CHANGES_REQUESTED** [review]
please use a non-admin account for screenshots

**Review by SammyOina — APPROVED** [review]

**Pull request merged**